### ISSUE TITLE
Android Crash in latest 5.4.3+4 - java.lang.NullPointerException: Attempt to invoke virtual method java.lang.String android.webkit.WebView.getUrl()

### ISSUE THREAD

**Issue by jeromeDms** [pichillilorenzo/flutter_inappwebview#1168]
- [x] I have read the [Get Started - Installation](https://inappwebview.dev/docs/get-started/installation/) section
- [x] I have read and done the [Get Started - Setup Android](https://inappwebview.dev/docs/get-started/setup-android/) section
- [x] I have read and done the [Get Started - Setup iOS](https://inappwebview.dev/docs/get-started/setup-ios/) section
- [x] I have already searched for the same problem

## Environment

| Technology           | Version       |
| -------------------- | ------------- |
| Flutter version      |      beta pre3         |
| Plugin version       |      5.4.3+4         |
| Android version      |     8.1.0          |
| iOS version          |               |
| Xcode version        |               |

Hi @pichillilorenzo 

I just received a crashLytics report with the following trace :
Less important than the previous bugs, just one report from one user for now, among several thousands users.

```
Fatal Exception: java.lang.NullPointerException: Attempt to invoke virtual method 'java.lang.String android.webkit.WebView.getUrl()' on a null object reference
       at com.pichillilorenzo.flutter_inappwebview.in_app_browser.InAppBrowserActivity.onCreateOptionsMenu(InAppBrowserActivity.java:57)
       at android.app.Activity.onCreatePanelMenu(Activity.java:3388)
       at androidx.fragment.app.FragmentActivity.onCreatePanelMenu(FragmentActivity.java:2)
       at androidx.appcompat.view.WindowCallbackWrapper.onCreatePanelMenu(WindowCallbackWrapper.java:2)
       at androidx.appcompat.app.AppCompatDelegateImpl$AppCompatWindowCallback.onCreatePanelMenu(AppCompatDelegateImpl.java:8)
       at androidx.appcompat.app.AppCompatDelegateImpl.preparePanel(AppCompatDelegateImpl.java:125)
       at androidx.appcompat.app.AppCompatDelegateImpl.doInvalidatePanelMenu(AppCompatDelegateImpl.java:61)
       at androidx.appcompat.app.AppCompatDelegateImpl$2.run(AppCompatDelegateImpl.java:22)
       at android.os.Handler.handleCallback(Handler.java:790)
       at android.os.Handler.dispatchMessage(Handler.java:99)
       at android.os.Looper.loop(Looper.java:164)
       at android.app.ActivityThread.main(ActivityThread.java:6494)
       at java.lang.reflect.Method.invoke(Method.java)
       at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:438)
       at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:807)
```

**Comment by pichillilorenzo** [comment]
Fixed with new release `5.4.3+5`

**Comment by pichillilorenzo** [comment]
Added another fix with the new release `5.4.3+7`
